pkg/controller/cloud/node_controller_test.go

```diff
@@ -557,21 +535,11 @@ func TestZoneInitialized(t *testing.T) {
 
 	cloudNodeController.AddCloudNode(fnh.Existing[0])
 
-	if len(fnh.UpdatedNodes) != 1 && fnh.UpdatedNodes[0].Name != "node0" {
-		t.Errorf("Node was not updated")
-	}
-
-	if len(fnh.UpdatedNodes[0].ObjectMeta.Labels) != 2 {
-		t.Errorf("Node label for Region and Zone were not set")
-	}
-
-	if fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneRegion] != "us-west" {
-		t.Errorf("Node Region not correctly updated")
-	}
-
-	if fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneFailureDomain] != "us-west-1a" {
-		t.Errorf("Node FailureDomain not correctly updated")
-	}
+	assert.Equal(t, 1, len(fnh.UpdatedNodes), "Node was not updated")
+	assert.Equal(t, "node0", fnh.UpdatedNodes[0].Name, "Node was not updated")
+	assert.Equal(t, 2, len(fnh.UpdatedNodes[0].ObjectMeta.Labels), "Node label for Region and Zone were not set")
+	assert.Equal(t, "us-west", fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneRegion], "Node Region not correctly updated")
+	assert.Equal(t, "us-west-1a", fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneFailureDomain], "Node FailureDomain not correctly updated")
```

Comment:
still some very long lines here and other files, please check the whole change

Reply:
Done. And checked other places too, I believe all assert lines can be viewed in one line now. Thanks!!